ginga/doc/download_doc.py

```diff
@@ -5,12 +5,17 @@
 import urllib
 
 from astropy.utils import minversion
-from astropy.utils.data import _find_pkg_data_path
 
 from ginga import toolkit
+from ginga.util.toolbox import ASTROPY_LT_4_3
 
 __all__ = ['get_doc']
 
+if ASTROPY_LT_4_3:
+    from astropy.utils.data import _find_pkg_data_path as get_pkg_data_path
+else:
+    from astropy.utils.data import get_pkg_data_path
+
 
```

Comment:
@pllim, any rationale for this instead of the usual:
```python
try:
    from astropy.utils.data import get_pkg_data_path
except ImportError:
    from astropy.utils.data import _find_pkg_data_path as get_pkg_data_path
```
?

Reply:
`except` is expensive and all usage with `astropy<4.3` (which is basically everyone not using the current dev because 4.3 is unreleased) will hit it. But maybe this is premature optimization. Also, having `if` check also makes finding and removing compatibility easier because the pattern is predictable compared to "human words" in in-line comments.

I'd like to keep the check as is, unless you have strong objection.

Thanks for the review!